src/listbox/ListboxOption.ts

```diff
@@ -14,15 +14,15 @@ export interface ListboxOptionProperties extends ThemedProperties {
 	label: DNode;
 	option: any;
 	selected?: boolean;
-	onClick?(option: any, index: number, key?: string | number): void;
+	onMouseDown?(option: any, index: number, key?: string | number): void;
```

Comment:
This change is going to need to be back-ported to version 5, so changing the event from `onclick` to `onmousedown` should really just be an internal detail - We can leave all the other references to `onClick` and leave the API as `onClick`.

Reply:
Cool got it, I'll just include this one line change then